src/clj/harja/kyselyt/kustannukset_kyselyt.clj

```diff
@@ -0,0 +1,8 @@
+(ns harja.kyselyt.kustannukset-kyselyt
+  (:require [jeesql.core :refer [defqueries]]))
+
+(defqueries "harja/kyselyt/kustannukset_kyselyt.sql"
+  {:positional? false})
+
+(declare hae-paikkaus-kustannukset hae-mpu-selitteet
+  tallenna-mpu-kustannus! )
```

Comment:
Näitä kyselyjä käytetään molemmissa, PPU, sekä MPU urakoissa:
```
-- name: hae-mpu-selitteet
-- name: tallenna-mpu-kustannus!
```

Pitäskö noi vielä uudelleennimetä, esim **hae-kustannus-selitteet**, **tallenna-yllapito-kustannus!** tms? 

Samoin HTTP palveluiden nimet, sekä funktiot. Näitä nyt on valitettavasti useita, koska tekohetkellä, ei ollut mitään ideaa, että näitä pitäisi käyttää muuallakin. 

Minua häiritsee hieman, että funktio nimet on MPU spesifiä, vaikka niitä käytetään myös PPU urakoissa, voi tulevaisuudessa aiheuttaa sekaannusta. Näiden korjailu ei kuitenkaan pitäisi olla kovin kummoista, tässä näitä ainakin on, jos jaksat korjata ja näät hyväksi niin korjaile, voi koodipesästä etsiä suoraan:

```
defn- hae-mpu-selitteet 
defn- tallenna-mpu-kustannus
gensym "mpu-kustannus"
hae-mpu-selitteet (autofill)
HaeMPUSelitteetEpaonnistui
HaeMPUSelitteetOnnistui
"MPU kustannusten apufunktiot"
:data-cy "mpu-kustannus-lisays"
:data-cy "tallena-mpu-kustannus"
:data-cy "mpu-kustannus-peruuta"
;; MPU kustannukset yhteenveto 
"MPU sopimustyyppisten urakoiden kustannukset"
deftest hae-mpu-selitteet-toimii
deftest tallenna-mpu-kustannus-toimii
-- MPU kustannukset 
mpu_kustannustyyppi_enum
```

Reply:
Päivitin nämä kanssa. Yllättävän vähän tuli ongelmila.